examples/set-namespace/advanced/README.md

```diff
@@ -41,7 +41,7 @@ configuration.
 
 Suppose we have a CRD with group `dev.example.com`, version `v1` and kind
 `MyResource`. We want the namespace to be set in field
```

Comment:
The above line `we embed the CRD in the kptfile` in this example...` seems inaccurate.